benchmarks/templates/benchmarks/success.html

```diff
@@ -2,11 +2,15 @@
 
 {% block content %}
 <section id="success" class="container center">
-	<p>Your submission was a success! Scores will be emailed to you whenever the tests complete.</p>
-  <p>Thank you for your participation!</p>
-  <br />
-  <p>To return to your profile page, click <a href="../profile">here</a>.</p>
-  <br />
-  <p>To submit another model, click <a href="../upload">here</a></p>
+  <h3>Your submission was a success! Scores will be emailed to you whenever the tests complete.</h3>
+  <h4>Thank you for your participation!</h4>
```

Comment:
let's not use headings for everything, use messages: https://bulma.io/documentation/components/message/